docker-app/qfieldcloud/core/views/users_views.py

```diff
@@ -1,17 +1,22 @@
-from django.contrib.auth import get_user_model
 from django.core.exceptions import ObjectDoesNotExist
 from drf_spectacular.utils import extend_schema, extend_schema_view
+
+
 from qfieldcloud.core import pagination, permissions_utils, querysets_utils
-from qfieldcloud.core.models import Organization, Project
+from qfieldcloud.core.models import (
+    Organization,
+    Project,
+)
```

Comment:
```suggestion
    Project
)
```

Can you remove all changes that are not relevant to this PR. This file should not be modified in this PR since you moved the code to the new file.

Reply:
Back to the initial state!